Agreements/Internal/TCNW_C.txt

```diff
@@ -7,10 +7,8 @@ COPX:EGHL:*:WCO:*:*:London TC NW:London AC Daventry:14000:*:^WCO
 
 COPX:EGLL:*:WOBUN:*:*:London TC NW:London AC Daventry:14000:*:^WOBUN
 COPX:EGLL:*:BUZAD:*:*:London TC NW:London AC Daventry:14000:*:^BUZAD
-COPX:EGLL:27L:*:*:*:London TC NW:London AC Daventry:14000:*:^UMLAT
-COPX:EGLL:27R:*:*:*:London TC NW:London AC Daventry:14000:*:^UMLAT
-COPX:EGLL:09L:*:*:*:London TC NW:London AC Daventry:14000:*:^ULTIB
-COPX:EGLL:09R:*:*:*:London TC NW:London AC Daventry:14000:*:^ULTIB
+COPX:EGLL:*:UMLAT:*:*:London TC NW:London AC Daventry:14000:*:^UMLAT
```

Comment:
UMLAT and ULTIB are further south than WOBUN/BUZAD - could this have an effect on the agreement displaying if they pass this point? Perhaps instead a FIXAFTER WELIN is better?

Reply:
I've had a more thorough look at this now and note the following:

- All UMLATs route via WOBUN and all ULTIBs via BUZAD
- All SRD routes via UMLAT/ULTIB route via WELIN
- L10 to DTY is _technically_ possible but not on any of the standard routes

So my suggestions are either:

- Collapse all UMLAT/ULTIB/WOBUN/BUZADs into one agreement:
`COPX:EGLL:*:*:WELIN:*:London TC NW:London AC Daventry:14000:*:^`; or
- Remove references to UMLAT/ULTIB and just use
`COPX:EGLL:*:WOBUN:*:*:London TC NW:London AC Daventry:14000:*:^
COPX:EGLL:*:BUZAD:*:*:London TC NW:London AC Daventry:14000:*:^`

Reply:
Or perhaps, we use FIX BEFORE as UMLAT/ULTIB instead of EGLL to respond to @cpawley's comments in #1974?

Reply:
Perhaps we go for...

COPX:EGLL:*:*:WELIN:*:London TC NW:London AC Daventry:14000:*:^WELIN
COPX:EGLL:*:UMLAT:*:*:London TC NW:London AC Daventry:14000:*:^WELIN
COPX:EGLL:*:ULTIB:*:*:London TC NW:London AC Daventry:14000:*:^WELIN
COPX:EGLL:*:WOBUN:*:*:London TC NW:London AC Daventry:14000:*:^WELIN
COPX:EGLL:*:BUZAD:*:*:London TC NW:London AC Daventry:14000:*:^WELIN

??

Reply:
Or put *:*:WELIN at the bottom as a catch-all. Either way, that would be my preference.

Reply:
Is there any need to add the WELIN agreement? We cover all the possible routes to WELIN from Heathrow so it's just adding an extra line that is redundant.

Reply:
I think having WELIN first is better, because the agreement will remain visible to the TC controller for longer in case the aircraft passes UMLAT/ULTIB whilst still on their frequency. (see position in relation to sector below) At least, that was how I understood the agreements worked, but maybe I'm wrong.

![image](https://user-images.githubusercontent.com/12017284/66723933-6d7bd000-ee17-11e9-9161-81792e784d7d.png)

Reply:
harry's logic is logical

Reply:
If we're going to have WELIN at the top anyway, my preference would be to remove the agreements for ULTIB, UMLAT, BUZAD and WOBUN. All UMLAT/ULTIB SRD routes outbound EGLL route via WELIN -- I assume the same was true for BUZAD/WOBUN before they were removed. Even in the highly unlikely event that a rereoute isn't done properly (i.e. UMLAT DCT WOBUN),, the agreement would still display.

Kill two birds with one stone, fix the issue and simplify, and (slightly!) reduce the size of, the sector file.

Reply:
now we're going round in circles :p 

> 
Or perhaps, we use FIX BEFORE as UMLAT/ULTIB instead of EGLL to respond to @cpawley's comments in #1974? I'm not sure if that could cause other issues though...

Reply:
My only concern with that was that that will match any aircraft going from TC to AC (as TCNW is usually bandboxed to TC) routeing via ULTIB (UMLAT is not a problem as it is the start of T418). Having said that I can't find anything from LTMA that routes along T420 via ULTIB except LL outbounds -- this could change though with SID truncations or other airspace changes, and we could cause ourselves headaches in the future. Seems like a needless complication when everything is funnelled via one waypoint (WELIN).

My preferred option is 
`COPX:EGLL:*:*:WELIN:*:London TC NW:London AC Daventry:14000:*:^ `

Reply:
Sorry for the confusion, I quoted the previous part of the conversation to refer to where we had proposed WELIN only - this is my concern. I think the agreement should have EGLL in the FIX BEFORE (because we want to avoid the issue you mention) and then at least WELIN, UMLAT and ULTIB to catch any weird flightplans - maybe WOBUN and BUZAD if we think they're common enough to retain.

Reply:
I'll change it to WELIN FIXAFTER, then UMLAT and ULTIB. WOBUN and BUZAD already warrant special attention to their flightplans by nature of the SID so I have faith that they will be correctly routed via WELIN. If we get reports to the contrary we can always readd.